components/Reference/index.tsx

```diff
@@ -258,7 +262,13 @@ const ReferenceTable = ({
                   <tr className="bg-indigo-50 dark:bg-black-600">
                     <td colSpan={colSpan}>
                       <DocRow
-                        itemDoc={itemDocs[opcodeOrAddress]}
+                        itemDoc={
+                          isAfterMerge
+                            ? opcodeOrAddress == '44'
+                              ? itemDocs['44_merge']
+                              : itemDocs[opcodeOrAddress]
+                            : itemDocs[opcodeOrAddress]
+                        }
```

Comment:
I'd extract this into useMemo method `itemDoc` with if/elses to make it easier to read.

Reply:
Struggling a bit with defining it as a `useMemo` since this function has the `opcodeOrAdderess` dependency.
If you could please suggest a change, I can commit it :)

Reply:
Ah sure, makes sense to use a callback here 👍 
Changed the code locally and I will test and merge everything tomorrow when we hopefully merge this PR.